### DISCUSSION TITLE
Is this available on crates.io?

### DISCUSSION THREAD

**Discussion by jinliu** [ChrisTitusTech/linutil#281] [Q&A] [answered]
So I can `cargo install` (and `cargo install-update`) it.

**Comment by koibtw** [comment, marked as answer]
#333

**Comment by sisyphos666** [comment]
is this a possible answer to the cargo prob?
https://github.com/nabijaczleweli/cargo-update

**Reply by koibtw** [reply]
Yes. [linutil_tui](https://crates.io/crates/linutil_tui) does work with cargo-update.
We also have an update command inside Linutil.
![image](https://github.com/user-attachments/assets/e2d86261-8f20-4946-bfc7-625ab0fc615c)
